README.md

```diff
@@ -16,53 +16,51 @@
 
 </div>
 
-## Usage
-
-First, go to the repository you want to check.
+## Configuration
 
+This tool looks for configuration in a `pyproject.toml`file in the root of the codebase being analysed.
 
-### [black](https://black.readthedocs.io/en/stable/)
+The default configuration values are:
 
-Check **src** directory and **setup.py** file with **black**
-```bash
-docker run --rm -v "${PWD}":/github/workspace plone/code-quality:latest check black src setup.py
+```toml
+[tool.plone-code-analysis]
+checkers = ["black", "flake8", "isort", "pyroma", "zpretty"]
+formatters = ["black", "isort", "zpretty"]
```

Comment:
Examples of the command format were missing from the README. Mainly saying that it is possible to execute the format individually for each formatter.

Reply:
Fixed